### ISSUE TITLE
disable Hardware touchscreen input and maintain scrcpy screen input

### ISSUE THREAD

**Issue by user4302** [Genymobile/scrcpy#2772]
- [✅] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.

**Is your feature request related to a problem? Please describe.**
this is a specific problem that is not related to android or scrcpy.
my device touchscreen is faulty and causes random clicks and swipes sometimes. 

**Describe the solution you'd like**
i would like an option to disable touchscreen inputs on the android, while maintaining inputs on scrcpy.
basically scrcpy becomes the primary, and only screen that the android has, while the adb connection is intact.

**Describe alternatives you've considered**
i used an app to disable the touchcreen, but it disables it on scrcpy as well.

**Additional context**
N/A

**Comment by rom1v** [comment]
I don't think it's possible.

**Comment by alimsk** [comment]
Hi, i had the same thing (my screen had ghost touch), solved it with `--turn-screen-off` option.

**Comment by ZinohJoe** [comment]
On the Pixel 4a updated to Android 12, the `--turn-screen-off` option does disable physical touchscreen input.

Strangely, the physical power button does not turn the screen back on, however the `MOD+SHIFT+O` shortcut does.

The links below show a recording I made showing the touchscreen not responding to physical input with `turn-screen-off`

Pixel 4a - Android 12 - turn-screen-off (wireless)
https://imgur.com/xwtlEVl

Pixel 4a - Android 12 - default (wireless)
https://imgur.com/dfCWgLu

**Comment by rom1v** [comment]
OK, thank you for the videos.

Scrcpy just changes the "power mode" to disable/enable physical screen. How the device behaves according to this setting is implementation-defined, and scrcpy could not have any impact on it AFAIK.

**Comment by user4302** [comment]
> Hi, i had the same thing (my screen had ghost touch), fixed it with `--turn-screen-off` option. it might be what you're looking for
no that doesnt turn off touch input, it just turns off the screen.

> On the Pixel 4a updated to Android 12, the `--turn-screen-off` option does disable physical touchscreen input.
thanks :o

> Scrcpy just changes the "power mode" to disable/enable physical screen. How the device behaves according to this setting is implementation-defined, and scrcpy could not have any impact on it AFAIK.
oh ok so it depends on the device then.

**Comment by CodeEleventh** [comment]
This is completely different, but...
Could "screen off and touch input on" be a feature? This gonna be a really good feature, especially to save power

**Comment by toms1232123123** [comment]
> * [✅] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.
> 
> **Is your feature request related to a problem? Please describe.** this is a specific problem that is not related to android or scrcpy. my device touchscreen is faulty and causes random clicks and swipes sometimes.
> 
> **Describe the solution you'd like** i would like an option to disable touchscreen inputs on the android, while maintaining inputs on scrcpy. basically scrcpy becomes the primary, and only screen that the android has, while the adb connection is intact.
> 
> **Describe alternatives you've considered** i used an app to disable the touchcreen, but it disables it on scrcpy as well.
> 
> **Additional context** N/A

hey may i get the name for the app
to turn off my toutchscreen cause i am only using scrpy for screen shere and i use a blue tooth mouse for my tablete
